<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="host_input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>host:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="user_input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>user:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="database_input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>database:</string>
     </property>
    </widget>
    <widget class="QLabel" name="wrong_connect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Wrong Conect!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Connect_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>571</width>
      <height>601</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="info_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>551</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>info:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="table_select">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Tables...</string>
     </property>
    </widget>
    <widget class="QLabel" name="table_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Table:</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>551</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="GraficWidget" name="grafic" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>551</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="select_column_1">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>540</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="select_column_label_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>540</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Select first column:</string>
     </property>
    </widget>
    <widget class="QLabel" name="select_column_label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>570</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Select second column:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="select_column_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>570</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>724</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraficWidget</class>
   <extends>QWidget</extends>
   <header>graficwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
